doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,124 @@
+.. _lsst.obs.decam.gen3:
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories. They assume the user has a recent weekly version of ``lsst_distrib`` setup.
+
+Local repository setup
+----------------------
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects).
+The latter step creates the ``DECam/calib`` collection.
+In the example here, ``REPO`` refers to the filepath you choose for the Butler repository.
+
+.. prompt:: bash
+
+   cd REPO
+   butler create REPO
+   butler register-instrument REPO lsst.obs.decam.DarkEnergyCamera
+   butler write-curated-calibrations REPO lsst.obs.decam.DarkEnergyCamera
+
+Check that this worked successfully by querying all the collections currently in the repo, e.g.,
```

Comment:
In your "check" sections, it would be good to have a comment on what the expected output should look like.

Reply:
you know... not nothing and not an error... ok.